### PR TITLE
Unchaining WIP-Release v0.1.33-alpha-build-00

### PR DESCRIPTION
**Pull request by SecUpwN**
Drumroll, please! Here it is, our probably most **awesome** release to date! A very special *THANKS* close to my heart flies out to @banjaxbanjo who worked his ass (and especially his nerves) off to finally make the much needed [major database overhaul]8https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/issues/215) happen! Dude, please always stay with us! :heart:

Another fine member I would like to mention here is @DJaeger. He has been digging deep into our app, found solutions for strange problems and did not hesitate to craft PRs wherever possible. We love you, too! Everyone who worked on our app the past 21 days: It's been a rough ride, but a real PLEASURE.

To sum it up: This is a real **SUMMER RELEASE**: Harder, better, faster and stronger. Stay safe everyone!

### CODE REVIEW

**Comment by banjaxbanjo** [comment]
@SecUpwN Cheers, Jesus that is one long change log. I'll be sticking around still alot of improvements needed to my PR's so plenty of work for me😃.

**Comment by SecUpwN** [comment]
@banjaxbanjo, ha, you bet! We haven't released to the public in quite some time, I hope users will love what we've created. But what you see here are just the commits - see our real [CHANGELOG](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/blob/HEAD/CHANGELOG.md).

**Comment by E3V3A** [comment]
he he, thanks guys, this is truly an awesome version... and seem very stable!
